packages/uswds-core/src/styles/mixins/helpers/add-link-styles.scss

```diff
@@ -1,6 +1,35 @@
 @use "sass:meta";
 @use "sass:list";
+@use "../../functions/utilities/color" as *;
```

Comment:
> [!TIP]
> Color tokens weren't working as expected because of missing util. Imported missing util, added docs, and created a unit test to ensure this works as expected.